src/error.rs

```diff
@@ -22,4 +22,6 @@ pub enum LurkError {
     Store(#[from] store::Error),
     #[error("Parser error: {0}")]
     Parser(#[from] parser::Error),
-}
+    #[error("Parser error: {0}")]
+    Provable(String),
```

Comment:
I'm not yet convinced this is the right name, but having the error type seems reasonable.

Reply:
I have no problem changing any of the error names. The ones I named (and some of the existing ones) definitely don't seem quite right to me either, but I had a hard time coming up with something better.

Reply:
Let me think about it. Provable might grow on me, but that word came from a context of remedial explanation of how Lurk works so grates as an enshrined description of what we mean here. We really just mean that this is a an actual Lurk-language-level error.